plugins/parsers/xpath/parser.go

```diff
@@ -38,17 +39,53 @@ type Parser struct {
 	DefaultTags         map[string]string `toml:"-"`
 	Log                 telegraf.Logger   `toml:"-"`
 
+	// Required for backward compatibility
+	ConfigsXML     []xpath.Config `toml:"xml" deprecated:"1.23.1;use 'xpath' instead"`
+	ConfigsJSON    []xpath.Config `toml:"xpath_json"`
+	ConfigsMsgPack []xpath.Config `toml:"xpath_msgpack"`
+	ConfigsProto   []xpath.Config `toml:"xpath_protobuf"`
```

Comment:
@srebhan Is there a reason why only the first one has this deprecation tag and the others manually  set a message in the Init method?

Reply:
Works for me.

Reply:
In that case, you are receiving the deprecation message twice when using `xpath_json` for example? That's not what I saw..